src/azure-cli/azure/cli/command_modules/storage/_params.py

```diff
@@ -146,6 +146,12 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
     timeout_type = CLIArgumentType(
         help='Request timeout in seconds. Applies to each call to the service.', type=int
     )
+    marker_type = CLIArgumentType(
+        help='A string value that identifies the portion of the list of containers to be '
+             'returned with the next listing operation. The operation returns the NextMarker value within '
+             'the response body if the listing operation did not return all containers remaining to be listed '
```

Comment:
Is there example that includes this argument?
Which value can the user can provide?

Reply:
discussed offline